docs/getting-started/getting-started-sql-snowflake.md

```diff
@@ -272,50 +284,121 @@ Clicking on any of the Gems shows these options.
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
 
-Once the aggregation expressions are specified, we can consider grouping by a particular column. Switch to the Group By tab, and type in n_nationkey .
+#### 4.4.3.3 Option to define the Aggregate logic
+
+Of course it's always possible to define transformations in the easy-to-use interface. Here let’s define the transformation logic for our **Aggregate** Gem, which will sum up the number of customers within each of the geographical locations and return a clean set of columns.
+
+![Aggregate definition](img/4-8-aggregate-definition.png)
+
+Within the **(1) Aggregate** tab define the following expressions on the **(2) Expressions** List. Most importantly, we create the `customers` column by counting customers using the `count(custkey)` expression. We also add two columns `name` and `nation_key`, that describe the selected location. Note, that to easily add columns you can simply click on them in the schema browser on the left side.
+
+Once the aggregation expressions are specified, we can consider grouping by a particular column. Switch to the Group By tab, and type in `nationkey`.
 
 Finally, we **(3) Rename** our Gem to `count_customers` and **(4) Save** it.
 
-#### 4.4.3 Interactively test
+#### 4.5 Interactively Test and Fix
 
-Now that our model is fully defined, with all the logic specified, it’s time to test it.
+Now that our model is fully defined, with all the logic specified, it’s time to make sure it works (and keeps working)!
+
+#### 4.5.1 Test
+
+You'll want to know how the data looks after each step, not just for the final table or view.
 
 ![Interactively run](img/4-9-interactively-run.png)
 
 Prophecy makes interactively testing the models incredibly easy! Simply click on the **(1) Play** button on any of the Gems and the model with all of it’s upstream dependencies will be executed. Once the model runs, the **(2) Result** icon appears. Click the Result icon to view a **(3) Sample** set of records.
 
+##### 4.5.2 FixIt
+
+If your model has some errors, perhaps introduced upstream without your knowledge, Prophecy's Copilot will suggest fixes automatically.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/17q9stsz6n?videoFoam=true" title="Drag and Drop Model Graph Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+The Fix with AI option appears at any point where you see an error message.
+
+## 5. Code view
```

Comment:
Just one video here is enough, let's shorten it

Reply:
This is a key value proposition and folks would just be getting introduced here, so I want to keep the first image as well as the video. Removed the coming-soon image.